packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss

```diff
@@ -0,0 +1,116 @@
+    }
+  }
+
+  &.gux-small {
+    width: var(--gse-ui-avatar-small-size);
+    height: var(--gse-ui-avatar-small-size);
+
+    & .gux-content {
+      width: calc(100% - var(--gse-ui-avatar-small-presenceRing-width) * 2);
+      height: calc(100% - var(--gse-ui-avatar-small-presenceRing-width) * 2);
+      margin: var(--gse-ui-avatar-small-presenceRing-width);
+      font-size: var(--gse-ui-avatar-small-initials-fontSize);
+    }
+  }
+
+  &.gux-xsmall {
+    width: var(--gse-ui-avatar-xsmall-size);
+    height: var(--gse-ui-avatar-xsmall-size);
+
+    & .gux-content {
+      width: calc(100% - var(--gse-ui-avatar-xsmall-presenceRing-width) * 2);
+      height: calc(100% - var(--gse-ui-avatar-xsmall-presenceRing-width) * 2);
+      margin: var(--gse-ui-avatar-xsmall-presenceRing-width);
+      font-size: var(--gse-ui-avatar-xsmall-initials-fontSize);
+    }
+  }
+}
+
+.gux-avatar-badge {
+  position: absolute;
+  top: calc(var(--gse-ui-avatar-content-large-border-width) * -1);
+  right: calc(var(--gse-ui-avatar-content-large-border-width) * -1);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: var(--gse-ui-avatar-large-badge-size);
+  height: var(--gse-ui-avatar-large-badge-size);
+  border-color: var(--gse-ui-avatar-content-default-border-color);
+  border-style: var(--gse-ui-avatar-content-default-border-style);
+  border-width: var(--gse-ui-avatar-content-large-border-width);
+  border-radius: 50%;
+
+  &.gux-medium {
+    top: calc(var(--gse-ui-avatar-content-default-border-width) * -1);
+    right: calc(var(--gse-ui-avatar-content-default-border-width) * -1);
+    width: var(--gse-ui-avatar-medium-badge-size);
+    height: var(--gse-ui-avatar-medium-badge-size);
+    border-width: var(--gse-ui-avatar-content-default-border-width);
+  }
+
+  &.gux-small,
+  &.gux-xsmall {
+    top: calc(var(--gse-ui-avatar-content-default-border-width) * -1);
+    right: calc(var(--gse-ui-avatar-content-default-border-width) * -1);
+    width: var(--gse-ui-avatar-small-badge-size);
+    height: var(--gse-ui-avatar-small-badge-size);
+    border-width: var(--gse-ui-avatar-content-default-border-width);
+  }
+}
+
+.gux-content {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  font-family: var(--gse-ui-avatar-large-initials-fontFamily);
+  font-size: var(--gse-ui-avatar-large-initials-fontSize);
+  font-weight: var(--gse-ui-avatar-large-initials-fontWeight);
+  color: var(--gse-ui-avatar-media-initialsForeground-inverse);
+  text-transform: uppercase;
+  background-color: var(--gse-ui-avatar-media-initialsBackground-default);
+  border: var(--gse-ui-avatar-content-default-border-width)
+    var(--gse-ui-avatar-content-default-border-style)
+    var(--gse-ui-avatar-content-default-border-color);
+  border-radius: 50%;
+}
+
+::slotted(img) {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
```

Comment:
Not sure it really matters but we typically put `::slotted` stuff after `:host` and before the internal css.

Reply:
done